ot2/octopus-full-ssh.py

```diff
@@ -0,0 +1,590 @@
+import sys
```

Comment:
Could you include a succinct module docstring here? The description for this PR is an excellent write-up and some of those bullet points would be great to include.

Reply:
Yes. I'll add descriptions for the other methods as well.